README.md

```diff
@@ -36,23 +63,24 @@ If you prefer to integrate the SDK using plain jar file, just download them from
 
```

Comment:
i suggest to remove the links to maven central here, as they are provided in the table at the end.

Reply:
makes sense - will change that.